src/dashboardGenerator.js

```diff
@@ -31,23 +31,25 @@ const SEVEN_DAYS = 60 * 60 * 24 * 7;
 
 const applyLimits = state => {
   // See: https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/cloudwatch_limits.html
-  const maxMetricsPerWidget = 100;
-  const maxMetricsPerServiceInAWidget = 2;
-  const maxServices = Math.floor(maxMetricsPerWidget / maxMetricsPerServiceInAWidget);
+  const maxMetricsPerDashboard = 500;
+  const maxMetricsPerServiceInAWidget = 1;
```

Comment:
1? surely `150`?

Or are the variables misnamed?
`totalServices`?
`totalMetricsPerWidget`?

Reply:
Hmm, I think the maths is a bit confusing here.

There are two limits:
maxMetricsPerWidget - 150.  
maxMetricsPerDashboard - 500

If a single widget is going to exceed the `perWidget` limit, then either:
- a new widget needs to be created
- the number of metrics in the widget is truncated

If the total number of metrics is going to be exceeded:
- either truncate the number of widgets
- truncate the number of metrics within a widget.

I'm not clear on which of the above code is trying to perform